mlrun/runtimes/kubejob.py

```diff
@@ -387,3 +388,79 @@ def func_to_pod(image, runtime, extra_env, command, args, workdir):
         ]
 
     return pod_spec
+
+
+class DatabricksRuntime(KubejobRuntime):
+    kind = "databricks-job"
+    _is_nested = True
+    _is_remote = True
+
+    @staticmethod
+    def get_enriched_code():
+        current_file = os.path.abspath(__file__)
+        current_dir = os.path.dirname(current_file)
+        databricks_runtime_wrap_path = os.path.join(
+            current_dir, "databricks_runtime_wrap.py"
+        )
+        wrap_code = "\n"
+        with open(databricks_runtime_wrap_path, "r") as databricks_runtime_wrap_file:
+            wrap_code += databricks_runtime_wrap_file.read()
+        return wrap_code
+
+    def _pre_run(self, runspec: RunObject, execution):
+        runspec.spec.parameters["internal_handler"] = runspec.spec.handler
+        runspec.spec.handler = "run_mlrun_databricks_job"
+
+    def run(
+        self,
+        runspec: Optional[
+            Union["mlrun.run.RunTemplate", "mlrun.run.RunObject", dict]
+        ] = None,
+        handler: Optional[Union[str, Callable]] = None,
+        name: Optional[str] = "",
+        project: Optional[str] = "",
+        params: Optional[dict] = None,
+        inputs: Optional[Dict[str, str]] = None,
+        out_path: Optional[str] = "",
+        workdir: Optional[str] = "",
+        artifact_path: Optional[str] = "",
+        watch: Optional[bool] = True,
+        schedule: Optional[Union[str, mlrun.common.schemas.ScheduleCronTrigger]] = None,
+        hyperparams: Optional[Dict[str, list]] = None,
+        hyper_param_options: Optional[HyperParamOptions] = None,
+        verbose: Optional[bool] = None,
+        scrape_metrics: Optional[bool] = None,
+        local: Optional[bool] = False,
+        local_code_path: Optional[str] = None,
+        auto_build: Optional[bool] = None,
+        param_file_secrets: Optional[Dict[str, str]] = None,
+        notifications: Optional[List[mlrun.model.Notification]] = None,
+        returns: Optional[List[Union[str, Dict[str, str]]]] = None,
+    ) -> RunObject:
+        if not project:
+            current_project = mlrun.get_current_project(silent=True)
+            if current_project:
+                project = current_project.name
```

Comment:
As discussed, I don't think you need to do this yourself, and this is handled by the `super()`, so in general I think this function doesn't need to be overridden if that's all it does.